src/main/java/org/jenkinsci/plugins/workflow/cps/steps/ParallelStep.java

```diff
@@ -46,7 +46,7 @@
     /**
      * All the sub-workflows as {@link Closure}s, keyed by their names.
      */
-    /*package*/ final Map<String,Closure> closures;
+    /*package*/ final transient Map<String,Closure> closures;
```

Comment:
Conversely: why would it be NON-transient if unserializable?   If there is not a solid reason than this is a tiny incidental bugfix, and I do not see why that is cause for complaint rather than praise.